ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -308,6 +308,8 @@ class ActionBar extends React.PureComponent {
     try {
       const interactiveTask = getAction(results.actions, 'create-interactive');
 
+      await taskcluster.getCredentials(currentRepo.tc_root_url);
```

Comment:
this doesn't actually update user.email- which is needed in the next steps.  I find that the taskcluster credentials are not username/email/etc. but auth keys for doing taskcluster actions.